src/components/shared/MembersList/MembersListItem.tsx

```diff
@@ -1,57 +1,45 @@
-import React, {
-  KeyboardEvent,
-  ReactNode,
-  useCallback,
-  useMemo,
-  useState,
-} from 'react';
-import { AddressZero } from '@ethersproject/constants';
+import React, { KeyboardEvent, ReactNode, useCallback, useMemo } from 'react';
 
-import { createAddress } from '~utils/web3';
-import UserMention from '~shared/UserMention';
+// import UserMention from '~shared/UserMention';
 import { ListGroupItem } from '~shared/ListGroup';
-import CopyableAddress from '~shared/CopyableAddress';
-import { AnyUser, Colony, useUser } from '~data/index';
-import { ENTER } from '~types';
-import HookedUserAvatar from '~users/HookedUserAvatar';
-import { getMainClasses } from '~utils/css';
 import MemberReputation from '~shared/MemberReputation';
+import UserAvatar from '~shared/UserAvatar';
+
+import { Contributor, Member } from '~types';
+import { ENTER } from '~types/index';
+import { getMainClasses } from '~utils/css';
+import { useColonyContext, useMobile } from '~hooks';
+import { DEFAULT_TOKEN_DECIMALS } from '~constants';
+import { notNull } from '~utils/arrays';
+
+import MemberActions from './Actions';
+import MemberInfo from './MemberInfo';
 
 import styles from './MembersListItem.css';
 
-interface Props<U> {
-  extraItemContent?: (user: U) => ReactNode;
-  colony: Colony;
-  onRowClick?: (user: U) => void;
+type User = Member['user'];
```

Comment:
You could use the `MemberUser` fragment defined in `~gql/fragments/user.graphql`